### PR TITLE
Drop php5 tests and make postgresql tests more universal

### PR DESCRIPTION
**Pull request by mimi1vx**
PHP5 isn't supported anymore
and we have two versions of PostgreSQL in active maintenance

verification run:
SLE12SP3 - http://quasar.suse.cz/tests/1093
SLE15       - http://quasar.suse.cz/tests/1094
SLE12SP4 - http://quasar.suse.cz/tests/1097

### CODE REVIEW

File: tests/console/postgresql_server.pm [hunk 1]
```diff
@@ -14,13 +14,13 @@ use base "consoletest";
...
 use utils;
-use version_utils 'sle_version_at_least';
+use version_utils 'is_sle';
 use apachetest;
 
 sub run {
     select_console 'root-console';
 
-    my $pgsql_server = sle_version_at_least('15') ? 'postgresql10-server' : 'postgresql96-server';
+    my $pgsql_server = (is_sle('15+') or is_sle('>12-sp3')) ? 'postgresql10-server' : 'postgresql96-server';
```
Comment (okurz) [inline tests/console/postgresql_server.pm]: will break Leap maintenance tests: https://openqa.opensuse.org/tests/781638#step/postgresql_server/2

Better phrase it the other way around: `(is_sle('<15') or is_leap('<15.0')) ? 'postgresql96-server' : 'postgresql10-server';`

Comment (mimi1vx) [reply tests/console/postgresql_server.pm]: `(is_sle('<15') or is_leap('<15.0')) ? 'postgresql96-server' : 'postgresql10-server'; ` is also wrong 

situation is more complicated with SLE12SP4 on way :D

Comment (coolo) [reply tests/console/postgresql_server.pm]: And we can't simply zypper in postgresql-server ?

File: tests/console/postgresql_server.pm [hunk 2]
```diff
@@ -14,13 +14,13 @@ use base "consoletest";
...
 use utils;
-use version_utils 'sle_version_at_least';
+use version_utils qw/is_sle is_leap/;
 use apachetest;
 
 sub run {
     select_console 'root-console';
 
-    my $pgsql_server = sle_version_at_least('15') ? 'postgresql10-server' : 'postgresql96-server';
+    my $pgsql_server = (is_sle('>=12-sp4') or is_leap('15.0+')) ? 'postgresql10-server' : 'postgresql96-server';
```
Comment (okurz) [inline tests/console/postgresql_server.pm]: Now you are leaving Tumbleweed behind again. I would also recommend same as coolo to install the virtual package postgresql-server

File: tests/console/postgresql_server.pm (hunk 1 above)
Comment (mimi1vx) [reply tests/console/postgresql_server.pm]: @coolo no, this isnt aviable on SLE12, on other hand postgresql10 is was released for all SLE12 servicepacks ..

File: tests/console/postgresql_server.pm (hunk 2 above)
Comment (coolo) [reply tests/console/postgresql_server.pm]: it's not virtual - it requires the default

Comment (mimi1vx) [reply tests/console/postgresql_server.pm]: `postgresql-server` package is on the SLE15+  real package which requires `postgresql-server-implementation`

on sle12 this package doesn't exist and is capability provided by postgresqlNM-server packages

**Pull request merged**